README.md

```diff
@@ -73,3 +75,13 @@ The server will start locally and on ngrok. The restaurants will not load becaus
 
 * [Here is the City's list of layers](http://maps.huntsvilleal.gov/arcgis/rest/services/Layers), each of which provides a REST endpoint from which we can query data. 
 * [Here's an example of a URL where you might query data](http://maps.huntsvilleal.gov/arcgis/rest/services/Layers/ParkLocations/MapServer/0/query).
+
+# Developers
+
+Instead of building the server and launching the server via make run, an alternative method to start the server is:
+>export GOOGLE_PLACES_API_KEY=supersecretkey
+>cd Frontier
+>npm start
+
+This eliminates the required build step allowing for faster debug cycles.
```

Comment:
Sure, sounds good.

On Mon, Jun 1, 2015 at 5:08 AM, Chad McElligott <notifications@github.com>
wrote:

> In README.md
> <https://github.com/codeforhuntsville/Frontier/pull/42#discussion_r31413335>
> :
>
> > @@ -73,3 +75,13 @@ The server will start locally and on ngrok. The restaurants will not load becaus
> >
> >  * [Here is the City's list of layers](http://maps.huntsvilleal.gov/arcgis/rest/services/Layers), each of which provides a REST endpoint from which we can query data.
> >  * [Here's an example of a URL where you might query data](http://maps.huntsvilleal.gov/arcgis/rest/services/Layers/ParkLocations/MapServer/0/query).
> > +
> > +# Developers
> > +
> > +Instead of building the server and launching the server via make run, an alternative method to start the server is:
> > +>export GOOGLE_PLACES_API_KEY=supersecretkey
> > +>cd Frontier
> > +>npm start
> > +
> > +This eliminates the required build step allowing for faster debug cycles.
>
> Since the goal of this pr is to learn the process of contributing, let's
> follow the process. Typically when you receive feedback on your pr you
> would update your branch (in this case, the master branch of your fork)
> with a new commit that addresses the issue the reviewer pointed out. That
> will update the pr automatically.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/codeforhuntsville/Frontier/pull/42/files#r31413335>.
>